README.md

```diff
@@ -1,2 +1,16 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+for each letter in each position in guess_word
+	likeness = 0
+	if letter is the same as in secret_word
+		likeness = likeness + 1 
+		if all letters are the same
+			print "correct!"
+		else
+			print "wrong..."
+			print likeness value
+     
+     
+ ![Flowchart](https://raw.githubusercontent.com/ac200905/comp110-worksheet-B/master/TerminalFlowchart%20-%20Page%201.png)
```

Comment:
Flowchart looks good. Only minor improvement would be to make it more explicit where the number of incorrect guesses in incremented.